crates/top/re_sdk/src/lib.rs

```diff
@@ -109,7 +109,7 @@ pub mod time {
 }
 pub use time::{IndexCell, Time, TimePoint, Timeline};
 
-pub use re_types_core::{
+pub use re_types::external::re_types_core::{
```

Comment:
nitnit: aren't all these e-exported on `re_types`? imho having to use `::external` internally is a bit weird

Reply:
Yeah, that was a leftover from before `re_types` times.